pages/linux/lsusb.md

```diff
@@ -0,0 +1,23 @@
+# lsusb
+
+> Display info about connected USB devices.
+
+- List all the USB ports available:
+
+`lsusb`
+
+- Hierarchy view, with each device's supported speed:
+
+`lsusb -t`
+
+- Get detailed info of a USB device:
+
+`lsusb -D {{device}}`
+
+- Show only devices with the specified vendor and product ID:
+
+`lsusb -d {{vendor}}:{{product}}`
+
+- List all mass storages:
```

Comment:
Perhaps renaming this example to `List verbose information about USB devices`
also, change the example to only `lsusb -v`.

Reply:
Thanks for the review @mfrw .

How about `List verbose information about connected mass storage devices`? Since I want to make the examples gradually more complex, and also I think 'verbose' is a common option that users may know already.

Reply:
Sounds good to me, @f4ww4z! Let's do that.

Reply:
Thanks @sbrl , I have updated the description in my new commit.

Reply:
@f4ww4z instead of mass storage, use USB devices, as this tool is not limited to storage.

Reply:
@mfrw Indeed it isn't. I think what @f4ww4z is getting at is that they'd like to show how you _could_ limit it to mass storage devices. The other examples above show that you can list all other types of devices too :-)